api/tests/routes/pro/get_stocks_test.py

```diff
@@ -29,7 +29,9 @@ def test_returns_an_event_stock(self, app):
             beginningDatetime=now,
             bookingLimitDatetime=now,
         )
-        bookings_factories.BookingFactory.create_batch(3, stock=stock)
+        bookings_factories.BookingFactory.create_batch(
+            3, stock=stock, user__deposit__expirationDate=datetime(year=2031, month=12, day=31)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏
> year=2031

Certes, mais c'est pas le cas, on aura plutôt today + 2 ans au mieux "dans la vraie vie"

Reply:
oui mais j'ai pas envie de mettre a jours l'année tout les ans du coup j'ai mis une date loingtaine. ça ne change rien en pratique il faut juste que ce soit superieur au now() de la db (qui n'est pas le now de python a cause de freeze gun)

Reply:
Et pourquoi pas `now5° + 10 ans` ?

Reply:
parceque ça dépend du now de la db et que le now de python est celui de freezegun du coup c'est compliqué